src/javascript/app/base/client.js

```diff
@@ -120,6 +120,9 @@ const Client = (() => {
         if (response.logout !== 1) return;
         removeCookies('login', 'loginid', 'loginid_list', 'email', 'residence', 'settings'); // backward compatibility
         removeCookies('reality_check', 'affiliate_token', 'affiliate_tracking', 'onfido_token', 'hide_guide');
+        // clear local storage
+        localStorage.removeItem('read_scam_message');
+        localStorage.removeItem('new_user_account');
```

Comment:
You can add these under `client.accounts` and let `ClientBase.clearAllAccounts();` remove it all.